### PR TITLE
Translated NewMainMenu, ReportForm, and various others.

### PR DESCRIPTION
**Pull request by DatPanDat**
# SCP:SL Translation PR Template

## Language Name
Thai

## State Type of change (New language, changed existing Etc.)
Updating translations to be up to date. (#377)

# Checklist:
- [ ] I have tested the translation changes ingame
- [x] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  (Exception to this is when edits has __only__ been made in-browser. If this is the case, also check this box.)
- [x] I have made sure that I'm **not** submitting a duplicate translation 
- [x] I have made sure to **not** included any advertisements/pointers to non-game stuff other than the translation author in the credits file
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation

### CODE REVIEW

File: Translations/th/NewMainMenu.txt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+SCP-079 - Toggle View Mode
+SCP-079 - Post-Processing
+ภาษา
+หน้าเมนูหลัก
+เชื่อมต่อ
+กลับ
+การเปลี่ยนแปลงนี้จะมีผลหลังการโหลดใหม่\nต้องการจะรีโหลดเมนูเลยไหม?
+ใช่
+ไม่
+Are you sure you want to quit?
```
Comment (Dat-Joseph) [inline Translations/th/NewMainMenu.txt]: Is this intentional? Just like the line 90 in the Main Menu file?

File: Translations/th/ReportForm.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+แจ้งผู้เล่น
+ผู้ต้องหา:\nไอดีของผู้เล่น:\nคำอธิบาย:
+ยกเลิก
+แจ้งข้อหา: แหกกฎของเซิร์ฟเวอร์
+แจ้งข้อหา:  Cheating or raiding.
```
Comment (Dat-Joseph) [inline Translations/th/ReportForm.txt]: same question for this :D

File: Translations/th/NewMainMenu.txt (hunk 1 above)
Comment (DatPanDat) [reply Translations/th/NewMainMenu.txt]: I might've missed that one, gonna fix it today.

File: Translations/th/ReportForm.txt (hunk 2 above)
Comment (DatPanDat) [reply Translations/th/ReportForm.txt]: I couldn't think of a word for that but I've got it now, will edit it today.

Comment (DatPanDat) [reply Translations/th/ReportForm.txt]: Translated. Raiding makes more sense in English so I'm keeping raiding in English.

File: Translations/th/NewMainMenu.txt (hunk 1 above)
Comment (DatPanDat) [reply Translations/th/NewMainMenu.txt]: Done.

**Pull request merged**